com.github.muriloventuroso.pdftricks.json

```diff
@@ -97,16 +103,26 @@
                 "--enable-zero-configuration"
             ],
             "sources": [{
-                    "type": "archive",
-                    "url": "https://imagemagick.org/archive/ImageMagick-7.1.0-61.tar.xz",
-                    "sha256": "4b4f9437666ef1a87df69b34bc8abdf7ad05d524c00dd1d63f6699e6ab1ed94e"
+                    "type": "git",
+                    "url": "https://github.com/ImageMagick/ImageMagick",
+                    "tag": "7.1.1-47"
                 },
                 {
                     "type": "patch",
                     "path": "patches/magick-policy.patch"
                 }
             ]
         },
+	{
+            "name": "libgee",
+            "buildsystem": "autotools",
+            "sources": [{
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/libgee/0.20/libgee-0.20.8.tar.xz",
+                    "sha256": "189815ac143d89867193b0c52b7dc31f3aa108a15f04d6b5dca2b6adfad0b0ee"
+                }
+            ]
+	},
```

Comment:
this is not properly indented. Don't make the json less readable.

Reply:
ok, did run `jq` now, so should be formatted.